src/utils/ScriptTransformer.ts

```diff
@@ -149,28 +147,43 @@ export function transformScript(filename: string, code: string, fileType: FileTy
     if (fileTypeFeature.isTypeScript) {
       presets.push("typescript");
     }
-    return babel.transform(code, { filename: filename, presets: presets }).code;
-  }
-  let parserConfig: ParserConfig;
-  if (fileTypeFeature.isTypeScript) {
-    parserConfig = {
-      syntax: "typescript",
-    };
-    if (fileTypeFeature.isReact) {
-      parserConfig.tsx = true;
+    const result = babel.transform(script.code, {
```

Comment:
We should probably go ahead and remove the forceBabelTransform branch at this point, instead of trying to maintain it after its useful life.

Reply:
:shrug:

I had actually originally implemented this only for babel (getting rid of the swc side of the branch), but it was trivial enough to add swc support as well.